share/spice/indego_bikeshare_locations/indego_bikeshare_locations.css

```diff
@@ -0,0 +1,63 @@
+.mapview-marker__icn.ddgsi-circle {
+  font-size: 6px;
+}
+
+.tile--indego_bikeshare_locations.tile--c--n {
+  width: 13em;
+  height: 13em;
+}
+
+.tile--indego_bikeshare_locations {
+    text-align: center;
+    height: 12em;
+}
+
+.tile--indego_bikeshare_locations .zci__header {
+    height: 2.5em;
+}
+
+.tile--indego_bikeshare_locations .side_by_side {
+    overflow: auto;
+    margin-left: auto;
+    margin-right: auto;
+    width: 100%;
+}
+
+.tile--indego_bikeshare_locations .side_by_side .left {
+    float: left;
+    border-right: 1px solid #E5E5E5;
+}
+
+.is-mobile .tile--indego_bikeshare_locations .side_by_side .left {
+    width: 50%;
+}
+
+.tile--citi_indego_bikeshare_locations .side_by_side .right {
+    float: right;
+}
+
+.is-mobile .tile--indego_bikeshare_locations .side_by_side .right {
+    width: 50%;
+}
+
+p.tx-clr--dk.tx--14 {
```

Comment:
Yup that's the final CSS. And yes, 31-42 are namespaced. As long as the rule has something that force it to only apply to our IA's div/tile/etc we're good.